packages/lib/RotatingLogs.ts

```diff
@@ -0,0 +1,51 @@
+import shim from './shim';
+import dayjs = require('dayjs');
+import { Stat } from './fs-driver-base';
+
+export interface RotationalLogs {
+	cleanActiveLogFile(): void;
+	deleNonActiveLogFiles(): void;
+}
+
+export default class RotatingLogs {
+
+	private logFilesDir = '';
+	private nonActiveLogFileMaximumDaysAge = 30;
```

Comment:
And 90 days

Reply:
And express it as milliseconds please, then you don't need to have code to convert between days and ms